service-registry/src/main/java/io/servicecomb/serviceregistry/definition/DefinitionConst.java

```diff
@@ -34,4 +34,6 @@
   String VERSION_RULE_LATEST = "latest";
 
   String VERSION_RULE_ALL = "0.0.0+";
+
+  String DEFAULT_REVISION = "0";
```

Comment:
there is no "DEFAULT_REVISION" logic, no need to define this.

Reply:
done